block-canvas/theme.json

```diff
@@ -188,10 +188,6 @@
                     "margin": {
                         "bottom": "0"
                     }
-                },
-                "typography": {
-                    "fontSize": "var(--wp--preset--font-size--large)",
```

Comment:
This was overriding the post title default sizes for H2s in the template, so I removed it.